src/app/configs/config-detail/config-detail.component.scss

```diff
@@ -0,0 +1,6 @@
+.mat-list-item {
+    font-size: 12px;
+}
```

Comment:
remove styling content.

 color should be taking care of by previous comment, and  overriding the font size of this single list isn't typically what we want to do since it will no longer match the rest of lists on the site.